serializr.js

```diff
@@ -167,8 +167,30 @@
             }
         }
 
+				// Ugly way to get the parameter names since they aren't easily retrievable via reflection
+				const STRIP_COMMENTS = /((\/\/.*$)|(\/\*[\s\S]*?\*\/))/mg;
+				const ARGUMENT_NAMES = /([^\s,]+)/g;
+				function getParamNames(func) {
```

Comment:
That's a good question, I haven't tried with any minimization.  Constructor params are a pretty special case, seems okay to require aliasing them to avoid the potential problem.